frontend/src/components/AnalyticsDrawer/ServiceAreaAnalytics.tsx

```diff
@@ -9,14 +9,27 @@ import './ServiceAreaAnalytics.css'
 
 export let ServiceAreaAnalytics = withStore(
   'adequacies',
-  'selectedServiceArea'
+  'selectedServiceArea',
+  'serviceAreas'
 )(({ store }) => {
 
-  let selectedServiceArea = store.get('selectedServiceArea')!
-  let adequacies = adequaciesFromServiceArea(selectedServiceArea, store)
+  let selectedServiceAreas = store.get('serviceAreas')
+  if (store.get('selectedServiceArea')) {
+    selectedServiceAreas = [store.get('selectedServiceArea')!]
+  }
+
+  let adequacies = adequaciesFromServiceArea(selectedServiceAreas, store)
 
   return <div className='ServiceAreaAnalytics'>
-    <AdequacyDoughnut serviceAreas={[store.get('selectedServiceArea')!]} />
+    <p className='Ellipsis Muted SmallFont'>{
+      store.get('uploadedServiceAreasFilename')
+        ? `Service Areas - ${store.get('uploadedServiceAreasFilename')}`
+        : `Service Areas - ${selectedServiceAreas.length.toLocaleString()} selected`
```

Comment:
Can you use a formatted?

Reply:
What do you mean?

Reply:
Sorry, typo (I'm doing code reviews from my phone).

I mean can you use our number formatted for consistency - https://github.com/bayesimpact/tds/blob/master/frontend/src/utils/formatters.ts#L18